examples/connectivity/plot_adhd_spheres.py

```diff
@@ -0,0 +1,76 @@
+"""
+Multi-subject connectome for ADHD
```

Comment:
Title is wrong. On top of this, I wouldn't mention ADHD in the title. I would focus on the fact that this is about extracting time series on a set of spheres.